virt/node_maintenance/virt-triggering-vm-failover-resolving-failed-node.adoc

```diff
@@ -1,26 +1,26 @@
 :_content-type: ASSEMBLY
 [id="virt-triggering-vm-failover-resolving-failed-node"]
-= Triggering virtual machine failover by resolving a failed node
+= Deleting a failed node to trigger virtual machine failover
 include::_attributes/common-attributes.adoc[]
 :context: virt-triggering-vm-failover-resolving-failed-node
 
 toc::[]
 
-If a node fails and xref:../../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[machine health checks] are not deployed on your cluster, virtual machines (VMs) with `RunStrategy: Always` configured are not automatically relocated to healthy nodes. To trigger VM failover, you must manually delete the `Node` object.
+If a node fails and xref:../../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[machine health checks] are not deployed on your cluster, virtual machines (VMs) with `runStrategy: Always` configured are not automatically relocated to healthy nodes. To trigger VM failover, you must manually delete the `Node` object.
 
 [NOTE]
 ====
 If you installed your cluster by using xref:../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[installer-provisioned infrastructure] and you properly configured machine health checks:
 
 * Failed nodes are automatically recycled.
-* Virtual machines with xref:../../virt/virtual_machines/virt-create-vms.adoc#virt-about-runstrategies-vms_virt-create-vms[`RunStrategy`] set to `Always` or `RerunOnFailure` are automatically scheduled on healthy nodes.
+* Virtual machines with xref:../../virt/node_maintenance/virt-about-node-maintenance.adoc#virt-about-runstrategies-vms_virt-about-node-maintenance[`runStrategy`] set to `Always` or `RerunOnFailure` are automatically scheduled on healthy nodes.
```

Comment:
Possibly out of scope for this PR, but IBM SG suggests avoiding lead-in sentences to a list that are not complete sentences (line 13 above).

Reply:
Yes, out of scope because someone else wrote thta note. But I guess I could clean it up.